code/src/test/pcgen/inttest/game_35e/pcGenGUIAliceTest.java

```diff
@@ -1,6 +1,28 @@
+/*
+ * pcGenGUIAliceTest.java
+ * Copyright 2015 (C) Andrew Maitland <amaitland@users.sourceforge.net>
```

Comment:
The original authors have to be the ones to release copyright to the project for these files. We'll have to go back through the history to find who they were and get their written release, or even better have them do a PR adding the headers. However for most it will be wither Tom or myself.

So, please remove the license headers you've added unless they are new files you've added (i.e. the msrd test classes you added are fine, even though they are derived form older work).